src/test/java/dk/cngroup/lentils/repository/FinalPlaceRepositoryTest.java

```diff
@@ -42,6 +47,30 @@ public void findByIdTest() {
         assertEquals(finalPlace.get(), finalPlaceOrig);
     }
 
+    @Test(expected = javax.validation.ConstraintViolationException.class)
+    public void finalPlaceWithEmptyTitleTest() {
+        FinalPlace finalPlace = new FinalPlace(TEST_EMPTY_TITLE, TEST_LOCATION, null);
+        repository.save(finalPlace);
+
+        assertEquals(0, repository.count());
+    }
+
+    @Test(expected = javax.validation.ConstraintViolationException.class)
+    public void finalPlaceWith256TitleTest() {
+        FinalPlace finalPlace = new FinalPlace(TEST_256_TITLE, TEST_LOCATION, null);
+        repository.save(finalPlace);
+
+        assertEquals(0, repository.count());
```

Comment:
Nejsem si jisty, ale tenhle assert se asi neprovede, kdyz je vyhozena vyjimka.
Je to tak? @mjanousek

Reply:
Neprovede. To same i v tech dalsich, kde pouzivas `expected =`. V JUnit 5 pak uz lze pouzit
```java
    assertThrows(NullPointerException.class, () -> {
        test.length();
    });
```
ale my pouzivame asi jen JUnit 4 (nejsem si jisty).

Reply:
JUnit 4 myslim. Je tam i ta starsi verze Spring Boot jeste.

Takze ty asserty klidne vyhod prosim.

Reply:
bez tech assertu ty testy padaji :O

Reply:
odstraneno